From 6078d82488fdf503391767f910f36cf9661ffbbc Mon Sep 17 00:00:00 2001
From: Alan Pevec <apevec@redhat.com>
Date: Thu, 23 Aug 2012 01:35:20 +0200
Subject: [PATCH] adjust egg info for Fedora

There is no egg info available for argparse on Fedora
so the requirement can't be satisfied.

FEDORA SPECIFIC - do not send upstream!
---
 tools/pip-requires |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/tools/pip-requires b/tools/pip-requires
index bbb4edf..69367c6 100644
--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -1,4 +1,3 @@
-argparse
 prettytable>=0.6,<0.7
 python-keystoneclient>=0.1.2,<0.2
 warlock<2
